mxcube3/ui/actions/general.js

```diff
@@ -100,6 +108,14 @@ export function getInitialStatus() {
         'Content-type': 'application/json'
       }
     });
+    const beamlineLimits = fetch('mxcube/api/v0.1/beamline', {
```

Comment:
I can add new route only for the limits, but thinking on it it is better to add the limits on the motors dictionary as well. Returning always {pos, state, limits} no matter if it is a diffractometer motor or a beamline movable.